src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -1,12 +1,23 @@
 package core.basesyntax;
 
+import java.time.DateTimeException;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
 import java.util.Optional;
 
 public class JavaDateTimeApi {
+    private static final String UKRAINE_OFFSET = "+02:00";
+    private static final DateTimeFormatter CUSTOM_DATE = DateTimeFormatter.ofPattern("d MMM yyyy");
+    private static final DateTimeFormatter SPECIAL_FORMAT = DateTimeFormatter
+            .ofPattern("dd LLLL yyyy HH:mm");
```

Comment:
```suggestion
    private static final DateTimeFormatter DATE_FORMATTER = DateTimeFormatter.ofPattern("d MMM yyyy");
    private static final DateTimeFormatter DATE_TIME_FORMATTER = DateTimeFormatter
            .ofPattern("dd LLLL yyyy HH:mm");
```
make these names informative